src/dataregistry/schema/schema.yaml

```diff
@@ -274,7 +274,21 @@ tables:
         foreign_key_table: "dataset"
         foreign_key_column: "dataset_id"
         description: "Dataset this alias is linked to"
-  
+      ref_alias_id:
+        type: "Integer"
+        foreign_key: True
+        foreign_key_schema: "self"
+        foreign_key_table: "dataset_alias"
+        foreign_key_column: "dataset_alias_id"
+        description: "Alias this alias is linked to"
+      access_api:
```

Comment:
I've maybe forgot why this is here, why does the dataset alias also need access_api and acess_api_configuration, do all aliases not eventually point to a dataset which holds these quantities?

Reply:
`access_api` is needed so that GCRCatalogs can filter on it when it initially queries the `dataset_alias` table to find out what's out there.
`access_api_configuration` is needed because the configuration file for an alias sometimes has more information than just the name of the thing it points to.  See for example https://github.com/LSSTDESC/gcr-catalogs/blob/master/GCRCatalogs/catalog_configs/buzzard.yaml. The keyword `include_in_default_catalog_list` is used by the code in some circumstances.